src/librustdoc/passes/lint/html_tags.rs

```diff
@@ -25,91 +25,85 @@ pub(crate) fn visit_item(cx: &DocContext<'_>, item: &Item) {
                 Some(sp) => sp,
                 None => item.attr_span(tcx),
             };
-            tcx.struct_span_lint_hir(
-                crate::lint::INVALID_HTML_TAGS,
-                hir_id,
-                sp,
-                msg.to_string(),
```

Comment:
remove this .to_string() here, rest is formatting

Reply:
https://github.com/rust-lang/rust/pull/117540/files?diff=unified&w=1 helps